schema.json

```diff
@@ -987,6 +987,33 @@
         },
         "person": {
             "additionalProperties": false,
+            "anyOf": [
+                {
+                    "required": [
+                        "alias"
+                    ]
+                },
+                {
+                    "required": [
+                        "email"
+                    ]
+                },
+                {
+                    "required": [
+                        "family-names"
+                    ]
+                },
+                {
+                    "required": [
+                        "given-names"
+                    ]
+                },
```

Comment:
I stumbled over `given-names`, thinking that it doesn't make self as identifying data by itself. I see why we would want to have it in the schema alongside `family-names`, as there may be a cultural dimension, and given names may be more suitable for identification than family names.

With either, I think a fix for #472 becomes highly relevant.

Reply:
Agree, in particular `family-names` and `given-names` lack a bit in uniqueness. To an extent, this is true for alias as well, because generally we don't intend that field to include a namespace / resolver, e.g. we'd write `alias: jspaaks`, not `alias: https://github.com/jspaaks`. But, none of that is a deal breaker for me.